wire-ios-request-strategy/WireRequestStrategy.xcodeproj/xcshareddata/xcschemes/WireRequestStrategy.xcscheme

```diff
@@ -37,6 +37,23 @@
             ReferencedContainer = "container:WireRequestStrategy.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
+      <AdditionalOptions>
+         <AdditionalOption
+            key = "MallocStackLogging"
+            value = ""
+            isEnabled = "YES">
+         </AdditionalOption>
+         <AdditionalOption
+            key = "PrefersMallocStackLoggingLite"
+            value = ""
+            isEnabled = "YES">
+         </AdditionalOption>
+         <AdditionalOption
+            key = "MallocScribble"
+            value = ""
+            isEnabled = "YES">
+         </AdditionalOption>
+      </AdditionalOptions>
```

Comment:
what does that do exactly? should it be enable for all other frameworks?

Reply:
I was debugging why some objects wasn't deallocated in a test, I'll revert this

Reply:
Reverted